src/qmflows/packages/SCM.py

```diff
@@ -211,39 +212,75 @@ def molecule(self, unit: str = 'bohr',
         return m
 
 
+class DFTB_Result(Result):
+    """Class providing access to PLAMS DFTBJob result results."""
+
+    def __init__(self, settings: Optional[Settings],
+                 molecule: Optional[plams.Molecule],
+                 job_name: str,
+                 dill_path: Union[None, str, os.PathLike] = None,
+                 plams_dir: Union[None, str, os.PathLike] = None,
+                 work_dir: Union[None, str, os.PathLike] = None,
+                 status: str = 'done',
+                 warnings: Optional[WarnMap] = None) -> None:
+        # Read available propiety parsers from a yaml file
+        super().__init__(settings, molecule, job_name, dill_path,
+                         plams_dir=plams_dir, properties=package_properties['dftb'],
+                         status=status, warnings=warnings)
+
+        if plams_dir is not None:
+            kf_filename = join(plams_dir, 'dftb.rkf')
+            # create a kf reader instance
+            self.kf = plams.KFFile(kf_filename)
+        else:
+            self.kf = None
+
+    @property
+    def molecule(self, unit: str = 'bohr',
```

Comment:
The ``Molecule.from_array()`` method might be of use here.
It takes an iterable of shape ``(n, 3)`` as argument and then updates all coordinates within the passed ``Molecule`` instance.

See https://github.com/SCM-NV/PLAMS/blob/e7067d56e750d89431b95a0beecb5a72bc1fdfc3/mol/molecule.py#L1502-L1509

Reply:
Oh nice! I am going to replace it, thanks!